esx_service/utils/error_code.py

```diff
@@ -12,18 +12,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
-""" Definiton of error message """
+""" Definiton of error code and error message """
+class ErrorCode:
+    # Tenant related error code start
+    VM_NOT_BELONG_TO_TENANT = 1
```

Comment:
I prefer to keep the name as it is since the VM is not "invalid", it just does not belong to the tenant.